DNN Platform/DotNetNuke.Web.Client/Controls/ClientResourceLoader.cs

```diff
@@ -51,6 +51,28 @@ protected override void OnPreRender(System.EventArgs e)
             if (AsyncPostBackHandlerEnabled)
             {
                 const string handlerScript = @"
+var loadScriptInSingleMode = function(){
+    var s = document.createElement( 'script' );
+    s.type = 'text/javascript';
+    s.src = window.dnnLoadScriptsInAjaxMode[0];
+    s.onload = window.dnnLoadScriptsInAjaxModeComplete;
+    s.onerror = window.dnnLoadScriptsInAjaxModeComplete;
+    document.body.appendChild(s);
+};
+
+var loadScriptInMultipleMode = function(){
+    for(var i = 0; i < window.dnnLoadScriptsInAjaxMode.length; i++){
+        var s = document.createElement( 'script' );
+        s.type = 'text/javascript';
+        s.src = window.dnnLoadScriptsInAjaxMode[i];
+        s.onload = window.dnnLoadScriptsInAjaxModeComplete;
+        s.onerror = window.dnnLoadScriptsInAjaxModeComplete;
+        document.body.appendChild(s);
+    }
+};
+
+var isFirefox = navigator.userAgent.toLowerCase().indexOf('firefox') > -1;
```

Comment:
If this requires user agent testing rather than feature testing, can we at least get some comments about what scenario causes us to behave differently in Firefox, so we can revisit that decision in the future?